build-ps/debian/percona-xtradb-cluster-server-5.7.preinst

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 get_pcount () {
-	PSCOUNT=$(ps -ef | grep "/usr/sbin/mysqld" | wc -l)
+	PSCOUNT=$(ps -C mysqld --no-headers | wc -l)
```

Comment:
I'm not sure if we should touch build-ps content. As per the process described in Conflu, let's leave this part for packaging team.